proposals/src/lib.rs

```diff
@@ -0,0 +1,106 @@
+#![no_std]
+#![deny(
+    missing_debug_implementations,
+    missing_copy_implementations,
+    trivial_casts,
+    trivial_numeric_casts,
+    unsafe_code,
+    unstable_features,
+    unused_import_braces,
+    unused_qualifications,
+    missing_docs,
+    rustdoc::broken_intra_doc_links,
+    unused_results,
+    clippy::all,
+    clippy::pedantic
+)]
+//! # Webb Protocol Proposals Specification
+//!
+//! A Reference Implementation of the Webb Protocol Proposals.
+//!
+//! ## Introduction
+//!
+//! Proposals are encoded in a simple binary format that describes the changes
+//! happened to a state of a system (for example, a smart contract) and we need
```

Comment:
**Sentence restructure suggestion:**

Proposals are encoded in a simple binary format that describes the state changes of a system (for example, a smart contract) and we need...